MainRepository/Program.cs

```diff
@@ -1,7 +1,44 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
-}
\ No newline at end of file
+﻿while (true) {
+   Console.Write ("Enter the number or press [e] to exit: ");

```

Comment:
There should be an input mode to select whether the output should be a word or roman number.